CLAUDE.md

```
@@ -0,0 +1,101 @@
+go run ./cmd/anti-tangent-mcp
+
+# Print version (uses VERSION file via -ldflags)
+go run ./cmd/anti-tangent-mcp --version
+
+# Local release artifacts dry-run
+goreleaser release --snapshot --clean --skip=publish
+```
+
+## Architecture
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)